crates/fee/src/lib.rs

```diff
@@ -434,30 +449,30 @@ impl<T: Config> Module<T> {
         Self::btc_for(amount, percentage)
     }
 
-    // Private functions internal to this pallet
```

Comment:
Some unnecessary code shift in this file (other than the pub exports), in general it's better to limit un-relevant changes - this PR is already quite large.

Reply:
I'm not sure what you mean. Besides the sketch code comments for the nomination feature, pub exports are the only thing that changed